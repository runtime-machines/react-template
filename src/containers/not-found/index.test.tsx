import * as React from 'react';
import { shallow } from 'enzyme';

import { NotFound } from '.';

describe('NotFound component', () => {
	it('should render successfully', () => {
		const tree = shallow(<NotFound />);

		expect(tree).toMatchSnapshot();
	});
});
