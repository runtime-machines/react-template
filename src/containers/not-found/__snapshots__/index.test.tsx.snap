// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should render successfully 1`] = `
<Wrapper>
  <div
    className="c-not-found"
  >
    404
    <br />
    Page not found
  </div>
</Wrapper>
`;
